### PR TITLE
watcher reconnect, deploy snapshot

### PR DESCRIPTION
**Pull request by jldec**

### CODE REVIEW

**Comment by jldec** [comment]
deploying this on GKE with the new reactor version from cc6ce067

File: ui/config/ui-service.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
     app: riff
     component: ui
 spec:
-  type: NodePort
+  type: LoadBalancer
```
Comment (markfisher) [inline ui/config/ui-service.yaml]: is this intentional?

Comment (jldec) [reply ui/config/ui-service.yaml]: vaccillating between types because i don’t know how to override on kubectl apply command line.

Comment (jldec) [reply ui/config/ui-service.yaml]: resolved in 41b74d1 using separate yaml for loadbalancer version

**Pull request merged**